plugins/hls-stan-plugin/src/Ide/Plugin/Stan.hs

```diff
@@ -162,24 +139,21 @@ rules recorder plId = do
 
               -- If envVar is set to 'False', stan will ignore all local and global .stan.toml files
               logWith recorder Debug (LogDebugStanEnvVars env)
-              seTomlFiles <- liftIO $ usedTomlFiles useDefConfig (stanArgsConfigFile stanArgs)
```

Comment:
I'm not sure about this, but seTomlFiles unused past this point.
There's also identical statement executed 5 lines above this and the seTomlFiles are just logged.

Reply:
This also slipped past me. The precious (repeated) statement is used to log which .stan.toml files were used to load the stan config. 

The binding should have a better name though. I think `stanTomlUsed` may be more useful.

Reply:
Ok, I renamed the other binding to `tomlsUsedByStan`